app/src/utils/api.ts

```diff
@@ -165,6 +164,22 @@ export const getDivisionData = async ({
   return res;
 };
 
+export const getCollectionsData = async ({
+  keyword = "",
+  sortID = "date-desc", // TODO: rename sortID TODO: store defaults as constants
```

Comment:
Make this a type, such as `type sortIds = "date-desc" | "date-asc" | ...;` Then you can define the set of values needed as constants. You should also reuse this in `collectionsSortOptions`, probably something like `const collectionsSortOptions: { [key in sortIds]: string } = {`.

This defines the `collectionsSortOptions` as an object of specific key values and string values (although you _could_ also give the string values a type).